blog/2021-05-16-terasaturday-14-spring-cleanup/index.md

```diff
@@ -52,7 +52,7 @@ It supports `aliases` to update existing labels instead of creating new ones and
 
 <div align="center">
     <a href="https://github.com/MovingBlocks/Terasology/labels" target="_blank" alt="Github labels of MovingBlocks/Terasology">
-    <img src="./github_labels.png" alt="Excerpt of our overhauled Github labels." width="80%"/>
```

Comment:
lets keep this as PNG to preserve transparency (they are roughly the same size or even smaller anyways).